Cargo.toml

```diff
@@ -3,10 +3,15 @@ name = "bfint"
 authors = ["Chudov Sergei <chudov42@gmail.com>"]
 description = "BrainFuck interpreter implemented in Rust."
 license = "MIT"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
@goDeni  I changed version here, because in previous PR with tests we have forgotten bump this version after backward incompatible changes

Reply:
I thought we don't do that yet